extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
nit: is this intended?

Reply:
The whiteline between the imports and license comment? We have it like that on most files I think